include/materials/NEMLStressInput.h

```diff
@@ -0,0 +1,24 @@
+/****************************************************************/
```

Comment:
I don't find the names of these classes to be terribly descriptive, but I'm struggling to come up with great names. `NEMLStressParamOverride`?

Reply:
There is only nemlstress class so the name is good.  I got rid of the seperate classes that did substitution.  Substitution is now done in the only remaining neml class as an optional parameter.